.github/workflows/build-test.yml

```diff
@@ -0,0 +1,51 @@
+name: BuildAndTestFunctions
```

Comment:
is it working at all? i mean running something?

Reply:
Yes - see https://github.com/streeva/jose-jwt/runs/1645966664?check_suite_focus=true.
Same will run on your parent (dvsekhvalnov/jose-jwt) repo one any merge of the PR,

Reply:
nice, thanks !